helm/cluster-cloud-director/templates/registry-secret.yaml

```
@@ -0,0 +1,8 @@
+{{- if .Values.registry.configure -}}
```

nit: there is a risk of `<.Values.registry.configure>: nil pointer evaluating interface {}.configure` if the values doesn't contain the `registry:` at all, and it can happen according to json schema.
```suggestion
{{- if and .Values.registry .Values.registry.configure -}}
```